management-server/src/main/kotlin/ut/isep/management/controller/InviteController.kt

```diff
@@ -77,7 +79,8 @@ class InviteController(val inviteService: InviteService) {
         @RequestBody inviteRequest: InviteCreateDTO
     ): ResponseEntity<URI> {
         return try {
-            val inviteUrl = inviteService.createInvite(inviteRequest)
+            val createdInvite = inviteCreateService.create(inviteRequest)
+            val inviteUrl = URI("https://localhost:8081/invite/${createdInvite.id}")
```

Comment:
Can't you do the base path (https://localhost:8081) in a .env and .env.development such that you don't have to change it between development and production

Reply:
Addressed in https://github.com/Alakind/ISEP/commit/59086e31403e901350d9403d31b8c04d073e22d1